client/packages/system/src/Stock/Components/InventoryAdjustment/InventoryAdjustmentModal.tsx

```diff
@@ -59,116 +45,42 @@ export const InventoryAdjustmentModal = ({
-              <Box display="flex" width={INPUT_WIDTH}>
-                <ReasonOptionsSearchInput
-                  disabled={draft.adjustment === 0}
-                  onChange={reason => setDraft(state => ({ ...state, reason }))}
-                  value={draft.reason}
-                  type={getReasonOptionTypes({
-                    isInventoryReduction,
-                    isVaccine: stockLine.item.isVaccine,
-                    isDispensary:
-                      store?.storeMode === StoreModeNodeType.Dispensary,
-                  })}
-                  width={INPUT_WIDTH}
-                />
-              </Box>
-            }
-          />
-          <StyledInputRow
-            label={t('label.on-hold')}
-            Input={<Checkbox checked={stockLine.onHold} disabled />}
-          />
-        </Box>
-        <Box
-          display="flex"
-          flexDirection="column"
-          gap={2}
-          paddingTop={2}
-          flex={1}
-        >
-          <StyledInputRow
-            label={t('label.pack-quantity')}
-            Input={<NumericTextDisplay value={stockLine.totalNumberOfPacks} />}
-          />
-          <StyledInputRow
-            label={t('label.available-packs')}
-            Input={
-              <NumericTextDisplay value={stockLine.availableNumberOfPacks} />
-            }
-          />
-          <StyledInputRow
-            label={t('label.adjust-by')}
-            Input={
-              <NumericTextInput
-                width={INPUT_WIDTH}
-                decimalLimit={2}
-                max={
-                  draft.adjustmentType === AdjustmentTypeInput.Reduction
-                    ? stockLine.totalNumberOfPacks
-                    : undefined
-                }
-                value={draft.adjustment}
-                onChange={value =>
-                  setDraft(state => ({
-                    ...state,
-                    adjustment: value ?? 0,
-                  }))
-                }
-              />
-            }
-          />
-          <StyledInputRow
-            label={t('label.new-pack-qty')}
-            Input={
-              <NumericTextInput
-                width={INPUT_WIDTH}
-                disabled={true}
-                value={
-                  stockLine.totalNumberOfPacks +
-                  (draft.adjustmentType === AdjustmentTypeInput.Addition
-                    ? draft.adjustment
-                    : -draft.adjustment)
-                }
-              />
-            }
-          />
-        </Box>
-      </Box>
+      <>
+        <ItemDetailAndStats stockLine={stockLine} variation={variation} />
+
+        <AdjustmentForm
+          isVaccine={stockLine.item.isVaccine}
+          draft={draft}
+          setDraft={setDraft}
+        />
+
+        {stockLine.availableNumberOfPacks + variation < 0 && (
+          <Alert severity="error" sx={{ marginRight: 0 }}>
+            {t('error.reduced-below-zero')}
+          </Alert>
+        )}
```

Comment:
Should restrict the width on this so it matches the main content a bit better?

<img width="1590" height="505" alt="Image" src="https://github.com/user-attachments/assets/f7dd63f5-ec48-4ffe-9c43-8add3d994308" />